src/System.Data.SqlClient/src/System/Data/SqlClient/SqlError.cs

```diff
@@ -90,5 +92,10 @@ internal int Win32ErrorCode
         {
             get { return _win32ErrorCode; }
         }
+
+        internal Exception Exception
```

Comment:
This is not needed. 
The property innerException of the base will be able to return this exception. We shouldn't be storing this in the SqlException.